expo-maps/build/GeoJson.d.ts

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { PatternItem } from './Common.types';
+/**
+ * GeoJson specific props.
+ */
+export declare type GeoJsonProps = {
+    /**
+     * JSON string containing GeoJSON
+     */
+    geoJsonString: string;
+    defaultStyle?: {
+        polygon?: {
+            fillColor?: string;
+            strokeColor?: string;
+            strokeWidth?: number;
+            strokeJointType?: 'bevel' | 'miter' | 'round';
+            strokePattern?: PatternItem[];
+        };
+        polyline?: {
+            color?: string;
+            pattern?: PatternItem[];
+        };
+        marker?: {
+            color?: string;
+            title?: string;
+            snippet?: string;
+        };
+    };
+};
+/**
+ * Internal JSON object for representing GeoJSON in Expo Maps library.
+ *
+ * See {@link GeoJsonProps} for more detail.
+ */
+export declare type GeoJsonObject = {
+    type: 'geojson';
+} & GeoJsonProps;
+/**
+ * GeoJSON component of Expo Maps library.
+ *
+ * Displays data provided in .json file.
+ * This component should be ExpoMap component child to work properly.
+ *
+ * See {@link GeoJsonProps} for more details.
+ *
+ * On Android you can style your features individually by specifing supported properties for given feature type:
+ *
+ * For polygon you can use:
+ * * fillColor
+ * * strokeColor
+ * * strokeWidth
+ * * strokeJointType
+ * * strokePattern
+ * props. Please check documentation for these five here {@link PolygonProps}.
+ *
+ * For polyline you can use:
+ * * color
+ * * pattern
+ * props. Please check documentation for these two here {@link PolylineProps}.
+ *
+ * For marker you can use:
+ * * color
+ * * title
+ * * snippet
+ * props. Please check documentation for these three here {@link MarkerProps}.
+ *
+ * @example
```

Comment:
I think there's no need to include an example geojson input file here but if you think it would be beneficial then leave it and feel free to resolve.

Reply:
I'm open to discuss it, but I assumed that it might be useful for the user to see the example and therefore be able to easier understand our functionalities 🤔 What do you think? Do you find it helpful?

Reply:
I thought there is a standard for GeoJson. You're right, the example will be helpful